Modules/_datetimemodule.c

```diff
@@ -6516,6 +6553,17 @@ PyInit__datetime(void)
     PyModule_AddIntMacro(m, MINYEAR);
     PyModule_AddIntMacro(m, MAXYEAR);
 
+    /* IsoCalendarDate */
+    if (!initialized) {
+        if (PyStructSequence_InitType2(&StructIsoCalendarDateType,
+                                       &struct_iso_calendar_date_desc) < 0) {
+            return NULL;
+        }
+        initialized = 1;
+    }
+    Py_INCREF((PyObject *) &StructIsoCalendarDateType);
+    PyModule_AddObject(m, "IsoCalendarDate", (PyObject *) &StructIsoCalendarDateType);
```

Comment:
I think we can drop this line - there's no particular need to expose this as a public, constructable object in the C module. You'll also need to add `IsoCalendarDate` into [this list of cleaned up symbols](https://github.com/python/cpython/blob/3f4db4a0bab073b768fae958e93288bd5d24eadd/Lib/datetime.py#L2513) so it isn't still exposed.